distribution/resources/bfsi-consent-management.xml

```diff
@@ -0,0 +1,84 @@
+        <AuthorizeSteps>
+            <Retrieve>
+                <Step class="org.wso2.bfsi.consent.management.extensions.authorize.impl.DefaultConsentRetrievalStep" priority="1"/>
+            </Retrieve>
+            <Persist>
+                <Step class="org.wso2.bfsi.consent.management.extensions.authorize.impl.DefaultConsentPersistStep" priority="1"/>
+            </Persist>
+        </AuthorizeSteps>
+        <Validation>
+            <Validator>org.wso2.bfsi.consent.management.extensions.validate.impl.DefaultConsentValidator</Validator>
+            <RequestSignatureAlias>wso2carbon</RequestSignatureAlias>
+            <JWTPayloadValidation>true</JWTPayloadValidation>
+        </Validation>
+        <AdminHandler>org.wso2.bfsi.consent.management.extensions.admin.impl.DefaultConsentAdminHandler</AdminHandler>
+        <ConsentAPICredentials>
+            <Username>admin</Username>
+            <Password>admin</Password>
+        </ConsentAPICredentials>
+        <Payments>
+            <MaximumInstructedAmount>1000</MaximumInstructedAmount>
+        </Payments>
+    </ConsentManagement>
+    <AuthenticationWebApp>
+        <ServletExtension>org.wso2.bfsi.consent.management.extensions.authservlet.impl.BFSIDefaultAuthServletImpl</ServletExtension>
+    </AuthenticationWebApp>
+    <PreserveConsentLink>true</PreserveConsentLink>
+    <Identity>
+        <ConsentIDClaimName>consent_id</ConsentIDClaimName>
+        <Extensions>
+            <RequestObjectValidator>org.wso2.bfsi.identity.extensions.auth.extensions.request.validator.DefaultBFSIRequestObjectValidator</RequestObjectValidator>
+            <ResponseTypeHandler>org.wso2.bfsi.identity.extensions.auth.extensions.response.handler.impl.BFSIDefaultResponseTypeHandlerImpl</ResponseTypeHandler>
+            <ClaimProvider>org.wso2.bfsi.identity.extensions.claims.BFSIDefaultClaimProvider</ClaimProvider>
+        </Extensions>
+        <ClientTransportCertAsHeaderEnabled>true</ClientTransportCertAsHeaderEnabled>
+        <TokenSubject>
+            <RemoveUserStoreDomainFromSubject>true</RemoveUserStoreDomainFromSubject>
+            <RemoveTenantDomainFromSubject>true</RemoveTenantDomainFromSubject>
+        </TokenSubject>
+    </Identity>
+    <HTTPConnectionPool>
+        <MaxConnections>2000</MaxConnections>
+        <MaxConnectionsPerRoute>1500</MaxConnectionsPerRoute>
+    </HTTPConnectionPool>
+    <Consent>
+        <Portal>
+            <ClientCredentials>
+                <ClientId>0NXYPMnh9mxSPCVjvvsVh17LWDca</ClientId>
+                <ClientSecret>pfVkfiEUvQqmZdYQuruauOs6SS0wd7yggkR2of58NXsa</ClientSecret>
+            </ClientCredentials>
+            <Params>
+                <IdentityServerBaseUrl>https://localhost:9443</IdentityServerBaseUrl>
+                <ApiManagerServerBaseUrl>https://localhost:8243</ApiManagerServerBaseUrl>
+            </Params>
+        </Portal>
+    </Consent>
+</Server>
```

Comment:
add new line

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/11/commits/a00b9b419db2aa1ba4198ff1a8f77e6bd545b1c4